src/marker.rs

```diff
@@ -77,6 +77,63 @@ impl MarkerListener {
         let loop_lines = blines.clone();
         let local_listener = tf_listener.clone();
 
+        let _sub = rosrust::subscribe(
+            &config.topic,
+            2,
+            move |msg: rosrust_msg::visualization_msgs::Marker| {
+                let mut _lines = loop_lines.write().unwrap();
+                _lines.clear();
+                if msg.action != 0 {
+                    return;
+                }
+                let res = &local_listener.clone().lookup_transform(
+                    &msg.header.frame_id,
+                    &static_frame.clone(),
+                    msg.header.stamp,
+                );
+                match &res {
+                    Ok(res) => res,
```

Comment:
Here it could make sense to print the error to rosout

Reply:
This is the 'default' implementation for transform from us.
rosout is not implemented: https://github.com/adnanademovic/rosrust/issues/138
Printing to the terminal is not an option, one would need to log and create an error widget or implement rosout ;)